Dalamud/Game/Addon/Lifecycle/AddonEvent.cs

```diff
@@ -6,67 +6,86 @@
 public enum AddonEvent
 {
     /// <summary>
-    /// Event that is fired before an addon begins it's setup process.
+    /// Event that is fired before an addon begins its setup process.
     /// </summary>
+    /// <remarks>Setup occurs when an addon is constructing its ui node lists, and registering event callbacks.</remarks>
```

Comment:
Please try, where possible, to keep to a single `<remarks>` element. You can use `<br />` for newlines.